### PR TITLE
Installer

### PR DESCRIPTION
**Pull request by mkrus**

### CODE REVIEW

**Comment by TimoBuskeKdab** [comment]
Hi, I found this file:
```
~\AppData\Local\Autodesk\3dsMax\2019 - 64bit\ENU\Plugin.UserSettings.ini
```
It contains the 3rd party dirs. In my case it looks like this:
```
[Directories]
Additional MAX plug-ins=C:\Program Files\Autodesk\3ds Max 2019\PlugIns\
Additional MAX plug-ins NVIDIA=C:\Program Files\Autodesk\3ds Max 2019\PlugIns\NVIDIA\Plugins\
Max Fluid plug-ins=C:\Program Files\Autodesk\3ds Max 2019\stdplugs\MaxFluid\bin\
Additional MAX plug-ins1=C:\Users\timo\dev\kuesa\github\kuesa-3dsmax\deploy\Kuesa\
[Help]
```
You can change it and 3DS Max will load your changes on the next startup.
Maybe it's worth adding the Kuesa install dir to [Directories] if not already existent to automate completely...

**Comment by mkrus** [comment]
ah right, I read about those ini files but could not find them on the system. I'll see if I can have installer update those, not sure it can

**Comment by lemirep** [comment]
Letting Timo or Nicolas confirm this works as I don't have 3DSMax set up

**Comment by TimoBuskeKdab** [comment]
> Letting Timo or Nicolas confirm this works as I don't have 3DSMax set up
Tested on my machine. Works.

**Comment by TimoBuskeKdab** [comment]
> Letting Timo or Nicolas confirm this works as I don't have 3DSMax set up

Tested on my machine. Works.

File: README.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ It is composed to 2 main parts:
   (such as wireframe export and layer handling)
   Note: Exporting wireframes is currently limited to tri faces.
 
+# Installing
+
+To install pre-built versions of the Kuesa plugins, please 
+download `InstallKuesaFor3DSMax-latest.exe` from http://bit.ly/KuesaFor3DSMax
+and run the installer.
+
+The plugins will be installed but 3dsmax will only be configured for the user
```
Comment (narnaud) [inline README.md]: It seems that the  is "3ds Max", if I trust the website. 3dsmax is ok as filename or project name, but in the readme we should use "3ds Max" everywhere.

Would be nice to fix also line 1 and 3.

File: README.md [hunk 2]
```diff
@@ -28,11 +40,14 @@ Please set the following environment variables, adjusting paths to match your in
 * Open Visual Studio, as administrator
 * Open `KuesaPlugins.sln`
 * If prompted, make sure is used `v140` build tools and `10.0.10586.0` SDK
-* Build 'Max2Babylon2019'. This will build the exporter and install it directly to 3DS Max.
+* Build 'Max2Babylon2019'. This will build the exporter and install it directly to 3dsmax.
```
Comment (narnaud) [inline README.md]: 3ds Max

File: README.md [hunk 3]
```diff
@@ -28,11 +40,14 @@ Please set the following environment variables, adjusting paths to match your in
...
 * Build 'KuesaLayers'. This will build the Kuesa Layers Manager into a subfolder of the deploy folder.
 
 ## Installing Kuesa Plugins For 3DS Max
 
+After building your own copy of the plugins, 3dsmax needs to be configured to use the latest versions.
```
Comment (narnaud) [inline README.md]: 3ds Max, also fix line 55 (no capitals)

**Comment by mkrus** [comment]
now with code signing

**Comment by narnaud** [comment]
Working nicely, nothing else to add

**Pull request merged**